TWLight/resources/models.py

```diff
@@ -190,10 +190,16 @@ class Meta:
         # Translators: In the administrator interface, this text is help text for a field where staff can link to a partner's Terms of Use.
         help_text=_("Link to terms of use. Required if users must agree to "
             "terms of use to get access; optional otherwise."))
-
-    description = models.TextField(blank=True, null=True,
-        # Translators: In the administrator interface, this text is help text for a field where staff can provide a description of a partner's available resources.
-        help_text=_("Optional description of this partner's resources."))
+            
+    short_description = models.TextField(max_length=1000, blank=True, 
```

Comment:
You should create a custom migration file to rename description to short_description, and then add long_description to save us having to copy all the existing descriptions over to short_description.

See https://gist.github.com/dhbradshaw/e2bdeb502b0d0d2acced